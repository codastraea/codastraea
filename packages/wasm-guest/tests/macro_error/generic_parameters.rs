use serpent_automation_wasm_guest::workflow;

#[workflow]
async fn counter<T>() {}

fn main() {}
