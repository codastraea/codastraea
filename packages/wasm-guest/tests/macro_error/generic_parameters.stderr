error: `workflow` functions should have no type parameters
 --> tests/macro_error/generic_parameters.rs:4:17
  |
4 | async fn counter<T>() {}
  |                 ^^^
